recipes/maxit/Makefile.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/Makefile b/Makefile
+index e821782..7282d03 100644
+--- a/Makefile
++++ b/Makefile
+@@ -22,9 +22,9 @@ LIBDIRS = common-v4.5 \
+ all:	compile
+ 
+ binary:	compile
+-	@sh -c './binary.sh'
+-	@./bin/DictToSdb -ddlFile ./data/ascii/mmcif_ddl.dic -dictFile ./data/ascii/mmcif_pdbx.dic -dictSdbFile mmcif_pdbx.sdb
+-	@mv mmcif_pdbx.sdb ./data/binary
```

Comment:
Because of memory shortage in CI environments, this process doesn't work for Linux platform.